tests/unit/options.js

```diff
@@ -2596,6 +2596,71 @@ singleGroups.lineNumber = function (test) {
   test.done();
 };
 
+singleGroups.unary = function (test) {
+  var code = [
+    "(-3).toString();",
```

Comment:
I tried to think of missing tests and although not directly related to the bugs, I can't find a test for `(a++).toString()` - which requires a bracket?
Otherwise I can't think of anything else sensible to test.